src/focus-visible.js

```diff
@@ -194,11 +167,23 @@ function init() {
     document.addEventListener('touchend', onInitialPointerMove);
   }
 
+  function removeInitialPointerMoveListeners() {
+    document.removeEventListener('mousemove', onInitialPointerMove);
+    document.removeEventListener('mousedown', onInitialPointerMove);
+    document.removeEventListener('mouseup', onInitialPointerMove);
+    document.removeEventListener('pointermove', onInitialPointerMove);
+    document.removeEventListener('pointerdown', onInitialPointerMove);
+    document.removeEventListener('pointerup', onInitialPointerMove);
+    document.removeEventListener('touchmove', onInitialPointerMove);
+    document.removeEventListener('touchstart', onInitialPointerMove);
+    document.removeEventListener('touchend', onInitialPointerMove);
+  }
+
   /**
    * When the polfyill first loads, assume the user is in keyboard modality.
-   * If any event is received from a pointing device (e.g mouse, pointer, touch), turn off
```

Comment:
Why remove this? It's not intended to be a complete list and may not be.

Reply:
cuz it was going to overflow :P i'll add it back